content/post/the-blacker-the-box.md

```diff
@@ -0,0 +1,74 @@
+
+1. Retention rates
+2. Purchase frequency
+3. Order values
+
+If you can model how long customers will remain customers, how often they order, and how much they spend, it turns out that you can build consistently good models of LTV. It *also* turns out, that there are lessons you can transfer from other businesses to your own. Retention curves, for every business I've examined, follow the same shape: there's a steep drop-off after acquisition that then flattens out into a steady-state low rate of churn for mature customers. *How steep* the initial drop is and *how long* until the curve begins to flatten out will be unique to your business, but the general pattern will pretty much always hold. 
+
+Because I know that this pattern holds consistently, and I can construct a formal model of how the three LTV ingredients play together, I can build a compelling model LTV that will generalize beyond my data set much better than a black-box model could. And, at the very least, because I'm able to reason about my formal model, I can explain to others the situations in which it *won't*  generalize very well, and help other decision-makers in the business take the appropriate amount of caution. The worst-case scenario for a black-box model is that it both fails to generalize and obfuscates that failure until it's too late (and lots of bad decisions have already been made).
+
+## In Practice
+
+Remember: all models are wrong, but some models are useful. When you're doing Data Science, make sure you're using the right tool for the job. 
+
+Sometimes it seems like ML practitioners and statisticians are warring tribes who do not understand why the other side uses techniques that are so clearly flawed[^6]. Really, as a discipline, we need to get better at recognizing the pros and cons of different approaches, and applying them judiciously to the problems we face.
+
+
+
+[^1]: "Correctness" could be True or False for binary predictions or some other quantification of error for regression-type problems.
+[^2]: I.e., it's not a one-shot high-stakes prediction for all-the-marbles.
+[^3]: I.e., made a purchase from an e-commerce store, downloaded an application, or performed whatever action the advertisement was trying to get you to take.
+[^4]: Nate Silver has made a whole career out of building really thoughtful, formal models that can outperform a pure ML model: https://fivethirtyeight.com/features/how-the-fivethirtyeight-senate-forecast-model-works/
+[^5]: Different flavors of these models at varying levels of sophistication will put slightly different spins on these metrics and how the model is fit, but at the most abstract level this pattern seems to hold.
+[^6]: I used to be one of these people, on the statistics side, who thought all ML was useless because the techniques did not have explanatory power. It took me a long time to realize that the people using ML *did not care about explanatory power*.
```

Comment:
I feel like the last line could live somewhere in the post or could be its own post!